west.yml

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2022 Legrand North America, LLC.
```

Comment:
Why it is done?
In zephyr project the MCUboot is just one of submodule. So zephyr-rtos manifest points given MCUboot version.

Reply:
This PR is to resolve MCUBoot issue #1410. 

The problem is the Zephyr port of MCUBoot does not allow me to configure and build an MCUBoot image from my proprietary workspace application module. This is different behavior from most(?) other Zephyr (sub)modules and prevents me from keeping all of my project configurations and builds in one repository. This refactoring aligns the Zephyr port with the Zephyr configuration and build system modularity and enables standard Zephyr module behavior.

Reply:
> This PR is to resolve MCUBoot issue #1410.
> 
> The problem is the Zephyr port of MCUBoot does not allow me to configure and build an MCUBoot image from my proprietary workspace application module. This is different behavior from most(?) other Zephyr (sub)modules and prevents me from keeping all of my project configurations and builds in one repository.

I believe this is solved by https://github.com/zephyrproject-rtos/zephyr/pull/51166

Reply:
Not entirely.  I am not building this as an extras image, and I expect to build and configure this in exactly the same way as other images.  Using sysbuild is having to use a different command (different behavior here), rather than having the _option_ to use a different command.

Reply:
@tejlmand I tried your suggestion but it does not solve my scenario.  It missed picking up boards/ configs within APPLICATION_CONFIG_DIR.